stage3/00-enable-access-point/01-run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+# Setup an static ip to wlan0
+cat files/dhcpcd.conf >> ${ROOTFS_DIR}/etc/dhcpcd.conf
```

Comment:
Are you sure that you want to append always and not to replace this file (and the others?)? (probably yes, just asking)

Reply:
So I have tried replacing instead of appending and it works.
dhcpcd.conf had extra configurations for dhcp client which we are not currently using
dnsmasq.conf had only one extra config for mac tagging of other raspberrys, which isn't useful neither.
hostapd.conf  was a completely new file anyway.
I like how it ends up after cleaning those. Thanks.